.github/workflows/build.yml

```diff
@@ -23,6 +23,9 @@ jobs:
   build:
     name: Build Plugin
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [ 20 ]
```

Comment:
Feels a bit unnecessary to make it into a matrix when we have no intention of using other node versions than the LTS.